app/Dockerfile

```
@@ -2,20 +2,15 @@ FROM python:3.7-alpine
 
 MAINTAINER faq@cryptic-game.net
 
-EXPOSE 80
-
 ENV MYSQL_PORT 3306
-ENV CROSS_ORIGIN True
-ENV DEBUG False
 
 WORKDIR /app
 
-ADD . /app/
-
-RUN apk add gcc musl-dev libffi-dev libressl-dev
+ADD ./Pipfile /app/
```

```suggestion
ADD . /app/
```

Is there a reason for this?

---

Yes, by adding the pipfile and installing the dependencies first we can make use of dockers layer caching. Otherwise we would always have to install the dependencies again which takes some time